x-pack/plugins/ml/public/application/timeseriesexplorer/_timeseriesexplorer_annotations.scss

```diff
@@ -56,6 +56,12 @@ $mlAnnotationRectDefaultFillOpacity: .05;
 
   fill: $euiColorFullShade;
   transition: fill $euiAnimSpeedFast;
+
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```

Comment:
we have `autoprefixer`, so you don't need this 
```suggestion
```

Reply:
Removed here [`baea0c3` (#98233)](https://github.com/elastic/kibana/pull/98233/commits/baea0c3354d2a6c0811b120f4c2f942c5b0b395f)